demisto_sdk/commands/content_graph/tests/parsers_and_models_test.py

```diff
@@ -1662,6 +1664,567 @@ def test_lazy_properties_in_the_model(self, mocker, pack):
         assert "python_version" in str(model)
 
 
+class TestFindContentType:
+    def test_integration_outside_content_path(self, git_repo):
+        """
+        Given:
+            - An integration YAML file outside the content repository path.
+        When:
+            - Running from_path on the integration YAML file.
+        Then:
+            - Verify that the returned Parser is IntegrationParser.
+        """
+        from demisto_sdk.commands.content_graph.parsers import IntegrationParser
+
+        integration_python_str = "integration.py"
+        integration_yaml_str = "integration.yml"
+        integration_yaml_path = git_repo.path / Path(integration_yaml_str)
+        with open(
+            "demisto_sdk/commands/content_graph/tests/test_data/integration.yml", "r"
+        ) as f:
+            content = f.read()
+        git_repo.make_file(file_name=integration_yaml_str, file_content=content)
+        git_repo.make_file(file_name=integration_python_str, file_content="Test")
+
+        assert isinstance(
```

Comment:
is there a generic way to ensure match() doesn't return true on any other content type?

Reply:
asking again :) It will be a great test if it's possible.

Reply:
we can add a test function for it:
iterate all `ContentItem`s, calling `match` on each of them and make sure that only the correct one is true